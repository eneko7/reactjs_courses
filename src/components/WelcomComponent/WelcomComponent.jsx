/* eslint-disable react/prefer-stateless-function */
import React, { Component } from 'react';

class WelcomComponent extends Component {
  render() {
    return (
      <div>Hello World - second method - React.Component</div>
    );
  }
}

export default WelcomComponent;
