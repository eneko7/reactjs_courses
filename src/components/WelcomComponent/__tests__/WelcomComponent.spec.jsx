import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import WelcomComponent from '../WelcomComponent';

const renderer = new ShallowRenderer();
renderer.render(<WelcomComponent />);
describe('WelcomComponent', () => {
  it('renders correctly', () => {
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
