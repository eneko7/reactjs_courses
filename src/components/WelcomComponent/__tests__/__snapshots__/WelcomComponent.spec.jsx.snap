// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomComponent renders correctly 1`] = `
<div>
  Hello World - second method - React.Component
</div>
`;
